assets/js/components/adminbar/AdminBarApp.js

```diff
@@ -81,37 +91,42 @@ export default function AdminBarApp() {
 						mdc-layout-grid__cell--align-middle
 					">
 						<div className="mdc-layout-grid__inner">
-							{ featureFlags.widgets.adminBar.enabled && (
+							{ zeroData && <AdminBarZeroData /> }
```

Comment:
Ideally we'd be using our `Grid`, `Row`, and `Cell` components here but that's for another issue.

```suggestion
							{ zeroData && (
								<div className="
									mdc-layout-grid__cell
									mdc-layout-grid__cell--span-12
								">
									<AdminBarZeroData />
								</div>
							) }
```